lib/client.js

```diff
@@ -4001,6 +4015,17 @@ module.exports.CRYPTO_ENABLED = CRYPTO_ENABLED;
  * @param {module:client~DeviceInfo} device information about the verified device
  */
 
+/**
+ * Fires whenever new user-scoped account_data is added.
+ * @event module:client~MatrixClient#"Room"
+ * @param {MatrixEvent} event The event describing the account_data just added
```

Comment:
This is technically incorrect. The sync.js code will emit an `accountData` event *before* it is added to the store (and hence gettable from `MatrixClient.getAccountData`. If this is intentional, please update this line to state that this describes the `account_data` *about to be* added. If this is not intentional, then please tweak `sync.js:611` to emit events *after* they have been added to the store.

There isn't a right answer as to whether you should emit before or after adding to the store, it greatly depends on what the purpose is of the thing you're emitting. This only works if you emit **before** for example:
```javascript
MatrixClient.on("accountData", function(event) {
  if (event.getType() === "one.to.one.room") {
    // find the old 1:1 room
    var oldEvent = client.getAccountData("one.to.one.room");
  }
});
```